gatsby-browser.js

```diff
@@ -7,7 +7,7 @@ function scrollToAnchor(location) {
   // Check for location so build does not fail
   const HEADER_OFFEST = 96;
   if (location && location.hash) {
-    const item = document.querySelector(`${location.hash}`);
+    const item = document.querySelector(`${CSS.escape(location.hash)}`);
```

Comment:
That looks legit, hovever you can just replace `querySelector` with `getElementById` and there is no neeed for `CSS.escape`